AppCenterDistribute/AppCenterDistribute/MSACDistribute.m

```diff
@@ -104,6 +104,7 @@ - (instancetype)init {
     // [MSACDistribute requestInstallInformationWith]
   }
                                   forService:kMSACServiceName];
+  [MSACUtility addMigrationClasses:@{@"MSReleaseDetails" : MSACReleaseDetails.self}];
```

Comment:
All initialization (including `MSAC_APP_CENTER_USER_DEFAULTS` should be inside `if` below)